.github/docs_pr_new_template.md

```diff
@@ -0,0 +1,16 @@
+## Description
+
+<!-- Describe the new documentation added in this PR -->
+
+## Checklist
```

Comment:
Are there any items in this list that we can condense? I find it a little intimidating right now.

Reply:
Consolidated in bb4691880555afd1a1d95512afe7ea2b6734e832